### PR TITLE
Added ability to supply an `Intervals` on a `Polyline`

### PR DESCRIPTION
**Pull request by dydkask**

### CODE REVIEW

**Comment by oguibueno** [comment]
Hi, can you provide more details about this Intervals support? Can you also create an example in the example project? thank you!

**Comment by dydkask** [comment]
The interval is a value that means the space between lines.
IOS_route

File: packages/atlas/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # atlas
 
+## v0.17.2 (2021-09-17)
```
Comment (oguibueno) [inline packages/atlas/CHANGELOG.md]:
```suggestion
## v0.17.2 (2021-10-13)
```

**Review by oguibueno — COMMENTED** [review]
Please update the date in the changelog.md so I can approve it. thank you

**Review by oguibueno — APPROVED** [review]

**Review by edsondiasalves — APPROVED** [review]

**Comment by oguibueno** [comment]
It's published @dydkask https://pub.dev/packages/atlas/versions/0.17.2

**Pull request merged**